src/components/v5/common/ActionSidebar/partials/ActionDescriptionMetadata/ActionDescriptionMetadata.tsx

```diff
@@ -0,0 +1,141 @@
+import { managePermissionsDescriptionMetadataGetter } from '../forms/ManagePermissionsForm/utils';
+import { manageTokensDescriptionMetadataGetter } from '../forms/ManageTokensForm/utils';
+import { mintTokenDescriptionMetadataGetter } from '../forms/MintTokenForm/utils';
+import { simplePaymentDescriptionMetadataGetter } from '../forms/SimplePaymentForm/utils';
+import { splitPaymentDescriptionMetadataGetter } from '../forms/SplitPaymentForm/utils';
+import { trasferFundsDescriptionMetadataGetter } from '../forms/TransferFundsForm/utils';
+import { unlockTokenDescriptionMetadataGetter } from '../forms/UnlockTokenForm/utils';
+import { upgradeColonyDescriptionMetadataGetter } from '../forms/UpgradeColonyForm/utils';
+
+const DESC_METADATA: Partial<Record<Action, DescriptionMetadataGetter>> = {
+  [ACTION.SIMPLE_PAYMENT]: simplePaymentDescriptionMetadataGetter,
+  [ACTION.ADVANCED_PAYMENT]: advancedPaymentDescriptionMetadataGetter,
+  [ACTION.SPLIT_PAYMENT]: splitPaymentDescriptionMetadataGetter,
+  [ACTION.TRANSFER_FUNDS]: trasferFundsDescriptionMetadataGetter,
+  [ACTION.MINT_TOKENS]: mintTokenDescriptionMetadataGetter,
+  [ACTION.UNLOCK_TOKEN]: unlockTokenDescriptionMetadataGetter,
+  [ACTION.MANAGE_TOKENS]: manageTokensDescriptionMetadataGetter,
+  [ACTION.EDIT_COLONY_DETAILS]: editColonyDetailsDescriptionMetadataGetter,
+  [ACTION.CREATE_NEW_TEAM]: createNewTeamDescriptionMetadataGetter,
+  [ACTION.EDIT_EXISTING_TEAM]: editTeamDescriptionMetadataGetter,
+  [ACTION.UPGRADE_COLONY_VERSION]: upgradeColonyDescriptionMetadataGetter,
+  [ACTION.ENTER_RECOVERY_MODE]: enterRecoveryModeDescriptionMetadataGetter,
+  [ACTION.CREATE_DECISION]: createDecisionDescriptionMetadataGetter,
+  [ACTION.MANAGE_PERMISSIONS]: managePermissionsDescriptionMetadataGetter,
+  [ACTION.MANAGE_COLONY_OBJECTIVES]:
+    manageColonyObjectivesDescriptionMetadataGetter,
+};
+
+const displayName =
+  'v5.common.ActionsContent.partials.ActionDescriptionMetadata';
+
+const ActionDescriptionMetadata = () => {
+  const formValues = useFormContext().getValues();
+  const selectedAction: Action | undefined = useWatch({
+    name: ACTION_TYPE_FIELD_NAME,
+  });
+  const apolloClient = useApolloClient();
+  const { user } = useAppContext();
+  const { colony } = useColonyContext();
+
+  if (!selectedAction || !colony || !user) {
+    return null;
+  }
+
+  const commonActionData: Omit<ColonyAction, 'type'> = {
+    initiatorAddress: user.walletAddress,
+    initiatorUser: user,
+    blockNumber: 0,
+    createdAt: new Date().toISOString(),
+    colony: {
+      ...colony,
+      nativeToken: {
+        ...colony.nativeToken,
+        nativeTokenDecimals: colony.nativeToken.decimals,
+        nativeTokenSymbol: colony.nativeToken.symbol,
+        tokenAddress: colony.nativeToken.tokenAddress,
+      },
+    },
+    colonyAddress: colony.colonyAddress,
+    showInActionsList: true,
+    transactionHash: ADDRESS_ZERO,
```

Comment:
Transaction hash !== address, they're different things altogether. We don't have a default fallback (as we do with the Null Address for addresses) for transactions